frontend/simple/views/Invite.vue

```diff
@@ -110,8 +116,12 @@ export default {
           contract.constructor.vuex.mutations[type](state, action.data)
         })
         let mailbox = await backend.latestHash(state.attributes.mailbox)
-        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId}, mailbox)
+        let date = new Date()
+        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId, sentDate: date.toString()}, mailbox)
```

Comment:
No, this date is to be shared between the group log and the mailbox invite and if it was set by the constructor it may differ.